src/components/pages/admin/adminRoutes.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { Admin } from '.'
+import { Route } from 'react-router'
+
+// http://stackoverflow.com/questions/35048738/react-router-import-routes
+// AssembliesRoutes.js
+// http://randycoulman.com/blog/2016/02/02/composing-routes-in-react-router/
+export const AdminRoutes = (
```

Comment:
filename could be something like:

admin/routes.tsx

Then the routes since we are exporting a const could be lower case, and here we could use the full admin name 

```
export const adminRoutes
```

The login behiind this:

- When I import is nice to have something consisten like

pages/admin/routes
pages/trainers/routes

- When I need to import the file itself I don't want to use aliases for the classes it's fine to use then specific name:

```
import {adminRoutes} from './pages/admin/routes'
import {trainersRoutes} from './pages/trainers/routes'
```

Do you agree?